google-cloud/gke/README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  <img src="https://secrethub.io/img/integrations/gke/github-banner.png?v1" alt="GKE + SecretHub" height="230">
+</p>
+<br/>
+
+<p align="center">
+  <a href="https://secrethub.io/docs/guides/gke/"><img alt="View Docs" src="https://secrethub.io/img/buttons/github/view-docs.png?v2" height="28" /></a>
+</p>
+<br/>
+
+This example uses Terraform to deploy the [SecretHub Demo App](https://secrethub.io/docs/start/getting-started/#consume) on a GKE cluster and provision it with the secrets it needs using the [SecretHub GCP Identity Provider](https://secrethub.io/docs/reference/gcp/). 
+
+## Prerequisites
+1. [Terraform](https://www.terraform.io/downloads.html) installed along with the [SecretHub Provider](https://secrethub.io/docs/guides/terraform/#install)
+1. Correctly configured [GCP Credentials](https://www.terraform.io/docs/providers/google/guides/provider_reference.html#full-reference)
+
+See [variables.tf](./variables.tf) for the variables you need to set.
+
+## Running the example
+
+Init the SecretHub demo repo with example values
+```
+secrethub demo init
+```
+
+To deploy the app on your cluster, run:
+```
+terraform init
+terraform apply
+```
+
+After launching it, forward the port to your local machine using `kubectl`:
+
+```
+kubectl port-forward deployment/demo-app 8080
+```
+
+To see the app running, visit [http://127.0.0.1:8080](http://127.0.0.1:8080).
```

Comment:
I'm seeing this is done consistent with the other examples, so it might be best if we address my comment in a separate PR:

It wasn't fully clear to me at a glance that this lists a couple of steps I need to take to run the example. We might want to clarify this somehow (preferably also when skimming the text). Some ways I could see this happening is adding numbers to the steps or adding transition words like "start with", "then", "finally" etc.

Reply:
Yep, I agree, but out of the scope of this PR I'd say.

Reply:
I moved `secrethub demo init` from the README to the `variables.tf` description, to keep this section even more to the point.